packages/tailwind-config/tailwind.config.mjs

```diff
@@ -84,9 +84,25 @@ export default {
     },
     container: {
       center: true,
-      padding: '16px',
+      screens: {
+        '2xl': '1608px',
+      },
+      padding: {
+        'DEFAULT': '1rem',
+        'sm': 0,
+        'lg': 0,
+        'xl': 0,
+        '2xl': 0,
+      },
     },
     extend: {
+      screens: {
+        // 'sm': '640px',
```

Comment:
just to make sure, do we want to keep this for the future?

Reply:
haha no, thanks :D I was just playing with it and forgot to remove